tox.ini

```diff
@@ -33,7 +33,7 @@ deps =
     isort
 commands =
     isort {[vars]all_path}
-    black {[vars]all_path}
+    black {[vars]all_path} --target-version=py311
```

Comment:
Should we be targeting python 3.10 since that is what comes with 22.04?

Reply:
Yes, you are correct. Updated, thanks!